app/src/lib/open-shell.ts

```diff
@@ -15,14 +15,20 @@ export function openShell(fullPath: string, shell?: string) {
     return spawn('START', [shell || 'cmd'], { shell: true, cwd: fullPath })
   }
 
+  if (__LINUX__) {
+    const commandArgs = ['--working-directory', fullPath]
+    return spawn('gnome-terminal', commandArgs, { shell: true })
+  }
+
   return fatalError('Unsupported OS')
 }
 
 export function isGitOnPath(): Promise<boolean> {
   // Modern versions of macOS ship with a Git shim that guides you through
   // the process of setting everything up. We trust this is available, so
   // don't worry about looking for it here.
-  if (__DARWIN__) {
+  // I decide linux user have git too :)
```

Comment:
> I only used ubuntu, which is bundled git by default.

It's not there on a fresh Ubuntu LTS install I did through Parallels:

<img width="762" src="https://user-images.githubusercontent.com/359239/28732146-316451dc-73ad-11e7-83e0-3be65e96f811.png">

And I upgraded the VM to Zesty but that broke the VM.

![](https://media.giphy.com/media/ZeNmLY6FISq4M/giphy.gif)